runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -1307,6 +1307,24 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 	 */
 	void fixupInternalLeafPointersAfterCopy(J9IndexableObject *destinationPtr, J9IndexableObject *sourcePtr);
 
+	/**
```

Comment:
this should be under a build flag, at least under DATA64
but even  better under OFFHEAP flag (in which case, call sites should be also under the same flag) 

@dmitripivkine, opinion?

Reply:
since in the current Offheap PR it's under DATA64, simplest is just to go that route, and later when everything is in, we can think about being more specific.